pkg/internal/test/test_doubles.go

```diff
@@ -47,7 +49,13 @@ func ExpectPayload(payloadAssert func(payload map[string]interface{}) bool) gock
 		}
 		var payload map[string]interface{}
 		err = json.Unmarshal(body, &payload)
-		payloadExpectations := payloadAssert(payload)
+		var payloadExpectations bool
+		for _, assertion := range payloadAssert {
+			payloadExpectations = assertion(payload)
+			if !payloadExpectations {
+				break
```

Comment:
I was wondering - would it make sense to make it a "soft assertion" instead?

Reply:
the same came to my mind I wanted to do it but forgot :-D

Reply:
Sure, I need this DSL :)

Reply:
will be part of #105